modules/ROOT/pages/guided-setup.adoc

```diff
@@ -25,12 +25,54 @@ This may be the US control plane, the EU control plane, the MuleSoft Government
 +
 If your Anypoint account has the organization administrator role, the guided setup tool can create the connected app for you.
 +
-Otherwise, create the connected app or ask an organization administrator to create it, then provide the app details so the guided setup can perform the authentication and authorization.
+Otherwise, create the connected app or ask an organization administrator to create it with these steps:
```

Comment:
```suggestion
Otherwise, ask an organization administrator to create the connected app for you, or perform these steps:
```

Reply:
Can you find a way to break up this task so that we don't have to go to three levels...especially since most of the user action is at the third level?

Reply:
Done. The only third level steps are now the 6 brief steps for creating a key pair. Also updated line 28 per your comment.